sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -570,13 +570,13 @@ RT::PiProgram ProgramManager::getBuiltPIProgram(
           "Unknown aspect " + std::to_string(static_cast<unsigned>(AspectNum)));
     }
   };
-#undef __SYCL_SPECIAL_ASPECT_DEPRECATED
+#undef __SYCL_ASPECT_DEPRECATED_ALIAS
 #undef __SYCL_ASPECT_DEPRECATED
 #undef __SYCL_ASPECT
 
   for (RTDeviceBinaryImage::PropertyRange::ConstIterator It : ARange) {
     auto KName = std::string_view((*It)->Name);
-    if (KName != "aspects")
+    if (KName != std::string_view("aspects"))
```

Comment:
nit: If done like this, this change is unnecessary.

Reply:
Can't compile with `"aspects"sv`

Reply:
What I mean is that my original suggestion was to eliminate named `KName` by using `string_view` on the string literal. If you keep `string_view` on the LHS, there is no need to transform the string literal.

Reply:
Okay, [51d9eab](https://github.com/intel/llvm/pull/6989/commits/51d9eab0cc21bc70085c2b7b75432cc92e3695aa)